docs/stable/serve/storage_aware_scheduling.md

```diff
@@ -0,0 +1,192 @@
+  serverlessllm/sllm-serve-worker
+```
+
+### Step 2: Start ServerlessLLM Serve with Storage Aware Scheduler
+
+1. Copy the hardware configuration file to the Ray head node.
+
+```bash
+docker cp hardware_config.json ray_head:/app/hardware_config.json
+```
+
+2. Start the ServerlessLLM serve with the storage aware scheduler.
+
+```bash
+docker exec ray_head sh -c "/opt/conda/bin/sllm-serve start --hardware-config /app/hardware_config.json"
+```
+
+### Step 3: Deploy Models with Placement Spec
+
+1. Create model deployment spec files.
+In this example, model "opt-2.7b" will be placed on server 0; while model "opt-1.3b" will be placed on server 1.
+```bash
+echo '{
+    "model": "opt-2.7b",
+    "backend": "transformers",
+    "num_gpus": 1,
+    "auto_scaling_config": {
+        "metric": "concurrency",
+        "target": 1,
+        "min_instances": 0,
+        "max_instances": 10
+    },
+    "placement_config": {
+        "target_nodes": ["0"]
+    },
+    "backend_config": {
+        "pretrained_model_name_or_path": "facebook/opt-2.7b",
+        "device_map": "auto",
+        "torch_dtype": "float16"
+    }
+}' > config-opt-2.7b.json
+echo '{
+    "model": "opt-1.3b",
+    "backend": "transformers",
+    "num_gpus": 1,
+    "auto_scaling_config": {
+        "metric": "concurrency",
+        "target": 1,
+        "min_instances": 0,
+        "max_instances": 10
+    },
+    "placement_config": {
+        "target_nodes": ["1"]
+    },
+    "backend_config": {
+        "pretrained_model_name_or_path": "facebook/opt-1.3b",
+        "device_map": "auto",
+        "torch_dtype": "float16"
+    }
+}' > config-opt-1.3b.json
+```
+
+> Note: Storage aware scheduling currently only supports "transformers" backend. Support for other backends will come soon.
+
+2. Deploy models with the placement spec files.
+
+```bash
+conda activate sllm
+export LLM_SERVER_URL=http://localhost:8343/
+
+sllm-cli deploy --config config-opt-2.7b.json
+sllm-cli deploy --config config-opt-1.3b.json
+```
+
+3. Verify the deployment.
+
+```bash
+curl http://localhost:8343/v1/chat/completions \
+-H "Content-Type: application/json" \
+-d '{
+        "model": "opt-2.7b",
+        "messages": [
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": "What is your name?"}
+        ]
+    }'
+
+curl http://localhost:8343/v1/chat/completions \
+-H "Content-Type: application/json" \
+-d '{
+        "model": "opt-1.3b",
+        "messages": [
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": "What is your name?"}
+        ]
+    }'
+```
+
+As shown in the log message, the model "opt-2.7b" is scheduled on server 0, while the model "opt-1.3b" is scheduled on server 1.
+```plaintext
+...
+(StorageAwareScheduler pid=1585) INFO 07-21 02:36:54 storage_aware_scheduler.py:119] Sorted scheduling options: [('0', 0.9877967834472656)]
```

Comment:
These logs didn't show when test using replay(more requests), I am not sure if it's ok.

Reply:
This is fine. We don't ask users to use `reply` here.